package.json

```diff
@@ -10,6 +10,7 @@
     "packages/*"
   ],
   "scripts": {
+    "add_tenant": "cd packages/server && node src/lib/addTenantScript",
```

Comment:
This is great, but might be better to have a reusable migration or seed file using knexjs. That way we gain predictable db configuration and initialization setup and also adding tenants will be shown in the migrations table.

Reply:
If I'm understanding you correctly - wouldn't that only create `tenants` in a test database?

I'm imagining us running this in the Render terminal shell and directly adding `tenants` to our production DB that way

Reply:
[Knex.js migration/seeds cli](http://knexjs.org/guide/migrations.html#migration-cli) can target any environment. It is standard to have seeds folder structure like
- seeds
  - development
  - staging
  - production
     - ...data

I was thinking that we would have the tenants and their agencies in csv files that a seed can then insert into a database. This will help us modify agencies easily and also gives us historical data of the agencies since it will be in git.

I think we should still add the script that you added since it will be useful for local dev and quick production changes if needed.

Reply:
I decided to keep this as a script, not a seed, because it requires user input and is intended to be invoked manually in one-off instances on prod. The tenants created there need not also be created in dev environment.